src/risedevtool/src/task/meta_node_service.rs

```diff
@@ -123,16 +123,18 @@ impl MetaNodeService {
                 is_persistent_meta_store = true;
 
                 cmd.arg("--backend")
-                    .arg("sql")
```

Comment:
Does the previous syntax still work? I am concerning about backward compatibility

Reply:
Yes, still works. The sql option is remained.